11ty/definitions/peanut-gallery.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Peanut Gallery
+slug: peanut-gallery
+flag:
+  level: avoid
+  text: Racist Term
+defined: true
+speech: noun
+reading:
+  - text: 'Is Peanut Gallery a racist term?'
+    href: http://www.todayifoundout.com/index.php/2015/03/origin-phrase-peanut-gallery/
+  - text: 'Dictionary.com'
+    href: https://www.dictionary.com/browse/peanut-gallery
+alt_words:
+  - disrupters
+  - hecklers
+---
+
+Used to refer to those giving unsolicited (and unvalued) advice.
```

Comment:
```suggestion
a group of people who are giving unsolicited (and unvalued) advice.
```

Reply:
I try to ensure that the definition doesn't presume what the "thing" is, which is why I added "a group of people." I would also suggest adding a bit about the origins of the term up here instead of in issues.

An example of where we do this is here: [Boogaloo](https://www.selfdefined.app/definitions/boogaloo/)